### ISSUE TITLE
Empty `assembly.fasta` due to filtering

### ISSUE THREAD

**Issue by schorlton** [fenderglass/ABruijn#393]
Looks like Flye filtered the only contig before output, so `assembly.fasta` was empty, and Flye exited 0. I wonder if this should be an error like when no disjointigs are generated?

```
[2021-06-02 18:45:07] DEBUG: UPath 1: 1 -> 2 -> -4
[2021-06-02 18:45:07] DEBUG: Final graph contains 1 egdes
[2021-06-02 18:45:07] DEBUG: Extending contigs into repeats
[2021-06-02 18:45:07] DEBUG: Covered 0 repetitive contigs
[2021-06-02 18:45:07] INFO: Generated 1 contigs
[2021-06-02 18:45:07] DEBUG: Writing FASTA
[2021-06-02 18:45:07] DEBUG: Writing Dot
[2021-06-02 18:45:07] DEBUG: Writing FASTA
[2021-06-02 18:45:07] DEBUG: Writing Gfa
[2021-06-02 18:45:07] DEBUG: Peak RAM usage: 0 Gb
-----------End assembly log------------
[2021-06-02 18:45:07] root: INFO: >>>STAGE: polishing
[2021-06-02 18:45:07] root: INFO: Polishing genome (1/1)
[2021-06-02 18:45:07] root: INFO: Running minimap2
[2021-06-02 18:45:07] root: INFO: Separating alignment into bubbles
[2021-06-02 18:45:12] root: DEBUG: Generated 4774 bubbles
[2021-06-02 18:45:12] root: DEBUG: Split 6 long bubbles
[2021-06-02 18:45:12] root: DEBUG: Skipped 0 empty bubbles
[2021-06-02 18:45:12] root: DEBUG: Skipped 0 bubbles with long branches
[2021-06-02 18:45:12] root: INFO: Alignment error rate: 0.104020
[2021-06-02 18:45:12] root: INFO: Correcting bubbles
[2021-06-02 18:45:15] root: DEBUG: Mean contig coverage: 2, selected threshold: 3
[2021-06-02 18:45:15] root: DEBUG: Filtered 1 contigs of total length 98689
[2021-06-02 18:45:15] root: DEBUG: Generating polished GFA
[2021-06-02 18:45:15] root: DEBUG: 1 sequences remained unpolished
[2021-06-02 18:45:15] root: INFO: >>>STAGE: finalize
[2021-06-02 18:45:15] root: DEBUG: ---Output dir contents:----
[2021-06-02 18:45:15] root: DEBUG: flye_output/
[2021-06-02 18:45:15] root: DEBUG:     365.0 B     assembly_graph.gv
[2021-06-02 18:45:15] root: DEBUG:     30.0 K      flye.log
[2021-06-02 18:45:15] root: DEBUG:     95.0 K      assembly_graph.gfa
[2021-06-02 18:45:15] root: DEBUG:     0.0 B       assembly.fasta
[2021-06-02 18:45:15] root: DEBUG:     92.0 B      params.json
[2021-06-02 18:45:15] root: DEBUG:     10-consensus/
[2021-06-02 18:45:15] root: DEBUG:         887.0 B     minimap.stderr
[2021-06-02 18:45:15] root: DEBUG:         129.0 K     consensus.fasta
[2021-06-02 18:45:15] root: DEBUG:         1.0 K       minimap.bam.bai
[2021-06-02 18:45:15] root: DEBUG:     00-assembly/
[2021-06-02 18:45:15] root: DEBUG:         124.0 K     draft_assembly.fasta
[2021-06-02 18:45:15] root: DEBUG:         29.0 B      draft_assembly.fasta.fai
[2021-06-02 18:45:15] root: DEBUG:     40-polishing/
[2021-06-02 18:45:15] root: DEBUG:         95.0 K      polished_edges.gfa
[2021-06-02 18:45:15] root: DEBUG:         26.0 B      filtered_stats.txt
[2021-06-02 18:45:15] root: DEBUG:         16.0 B      edges_aln.bam.bai
[2021-06-02 18:45:15] root: DEBUG:         0.0 B       filtered_contigs.fasta
[2021-06-02 18:45:15] root: DEBUG:         1.0 K       minimap_1.bam.bai
[2021-06-02 18:45:15] root: DEBUG:         881.0 B     minimap.stderr
[2021-06-02 18:45:15] root: DEBUG:         43.0 B      contigs_stats.txt
[2021-06-02 18:45:15] root: DEBUG:     20-repeat/
[2021-06-02 18:45:15] root: DEBUG:         96.0 K      graph_before_rr.fasta
[2021-06-02 18:45:15] root: DEBUG:         72.0 K      read_alignment_dump
[2021-06-02 18:45:15] root: DEBUG:         129.0 K     repeat_graph_edges.fasta
[2021-06-02 18:45:15] root: DEBUG:         589.0 B     graph_after_rr.gv
[2021-06-02 18:45:15] root: DEBUG:         501.0 B     graph_before_rr.gv
[2021-06-02 18:45:15] root: DEBUG:         714.0 B     repeat_graph_dump
[2021-06-02 18:45:15] root: DEBUG:     30-contigger/
[2021-06-02 18:45:15] root: DEBUG:         365.0 B     graph_final.gv
[2021-06-02 18:45:15] root: DEBUG:         96.0 K      graph_final.fasta
[2021-06-02 18:45:15] root: DEBUG:         0.0 B       scaffolds_links.txt
[2021-06-02 18:45:15] root: DEBUG:         96.0 K      contigs.fasta
[2021-06-02 18:45:15] root: DEBUG:         95.0 K      graph_final.gfa
[2021-06-02 18:45:15] root: DEBUG:         110.0 B     contigs_stats.txt
[2021-06-02 18:45:15] root: DEBUG:         24.0 B      contigs.fasta.fai
[2021-06-02 18:45:15] root: DEBUG:     22-plasmids/
[2021-06-02 18:45:15] root: DEBUG:         266.0 B     minimap_1.bam
[2021-06-02 18:45:15] root: DEBUG:         129.0 K     repeat_graph_edges.fasta
[2021-06-02 18:45:15] root: DEBUG:         0.0 B       plasmids_raw.fasta
[2021-06-02 18:45:15] root: DEBUG:         16.0 B      minimap_1.bam.bai
[2021-06-02 18:45:15] root: DEBUG:         879.0 B     minimap.stderr
[2021-06-02 18:45:15] root: DEBUG:         0.0 B       bubbles_1.fasta
[2021-06-02 18:45:15] root: DEBUG:         0.0 B       polished_1.fasta
[2021-06-02 18:45:15] root: DEBUG:         26.0 B      contigs_stats.txt
[2021-06-02 18:45:15] root: DEBUG:         714.0 B     repeat_graph_dump
[2021-06-02 18:45:15] root: DEBUG:     21-trestle/
[2021-06-02 18:45:15] root: DEBUG:         168.0 B     trestle_summary.txt
[2021-06-02 18:45:15] root: DEBUG:         0.0 B       repeats_dump
[2021-06-02 18:45:15] root: DEBUG:         129.0 K     repeat_graph_edges.fasta
[2021-06-02 18:45:15] root: DEBUG:         0.0 B       resolved_copies.fasta
[2021-06-02 18:45:15] root: DEBUG:         714.0 B     repeat_graph_dump
[2021-06-02 18:45:15] root: DEBUG: --------------------------
```

**Comment by mikolmogorov** [comment]
Good point - I agree that Flye should return 1 in this case.

**Comment by mikolmogorov** [comment]
Fixed in 3ca19a6
